src/Model/ROI.py

```diff
@@ -715,9 +717,63 @@ def transform_rois_contours(axial_rois_contours):
                         sagittal_rois_contours[name][contour[i][0]] = [[]]
                         sagittal_rois_contours[name][contour[i][0]][0]\
                             .append([contour[i][1], slice_ids[slice_id]])
+
+    coronal_rois_contours = convert_coordinates_map_to_polygon_of_rois(
+        coronal_rois_contours)
+    sagittal_rois_contours = convert_coordinates_map_to_polygon_of_rois(
+        sagittal_rois_contours)
     return coronal_rois_contours, sagittal_rois_contours
 
 
+def convert_coordinates_map_to_polygon_of_rois(contours_map):
+    polygon_dict = {}
```

Comment:
Can you please add docstrings to this function?

Reply:
done